### DISCUSSION TITLE
ICO on Kadena

### DISCUSSION THREAD

**Discussion by thakurami** [kadena-community/docs#17] [Q&A]
Hello,

This is Amit. We are doing an ICO project on Kadena. 

We are writing the smart contract using PACT and then plan to test on "testnet" first.  But for buying KDA it seems we need to use Zelcore/Coinmetro. Is that correct? Or are there any simpler way?

Thanks and best regards
Amit

**Comment by trendzetter** [comment]
these topics are covered on the docs pages:
https://docs.kadena.io/basics/exchanges
https://docs.kadena.io/basics/wallets
